plugins/modules/iam_user.py

```diff
@@ -273,15 +281,31 @@ def create_or_update_login_profile(connection, module):
     return True, retval
 
 
-def delete_login_profile(connection, module):
-
-    user_params = dict()
-    user_params['UserName'] = module.params.get('name')
+def delete_login_profile(connection, module, user_name, wait):
```

Comment:
You probably don't need wait parameter here, you can simply get it using module.params.get('wait').

Reply:
yeah, I guess not. I felt it would be better to retrieve it once rather than again and again but it doesn't matter I supposed

Reply:
updated to make the method names more consistent - previously, some had 'user' as a parameter and some didnt